content/blog/graphql-as-a-gateway/index.md

```diff
@@ -0,0 +1,200 @@
+---
+title: GraphQL feat API Gateway
+date: '2019-07-01T08:00:00Z'
+cover: './cover.jpg'
+---
+
+![Photo by Christian Stahl on Unsplash](cover.jpg)
+
+GraphQL, REST, GRPC, Thrift... Have you ever imagined how to stick these together, in a micro-services architecture, and expose to the world? There are some common ways to do it, such as using Nginx or Kong, but an alternative way to do this is by using GraphQL in front of all services.
+
+## ⏩ API Gateway pattern -- a quick introduction
```

Comment:
For each item on the responsibilities list, add some detail about it on the post (most likely on the section before Caveats)